install_config/topics/full_rhv_inventory.adoc

```diff
@@ -58,59 +55,32 @@ openshift_node_local_quota_per_fsgroup=512Mi <2>
 openshift_use_system_containers=False
 
 [masters]
-master0.example.com
-master1.example.com
-master2.example.com
+# dynamically set by openshift_ovirt role
 
 [etcd]
-master0.example.com
-master1.example.com
-master2.example.com
+# dynamically set by openshift_ovirt role
 
 [infras]
-infra0.example.com
-infra1.example.com
-infra2.example.com
-
-[glusterfs_registry]
-infra0.example.com glusterfs_devices="['/dev/vdd']"
-infra1.example.com glusterfs_devices="['/dev/vdd']"
-infra2.example.com glusterfs_devices="['/dev/vdd']"
+# dynamically set by openshift_ovirt role
 
 [lb]
-lb.example.com
+# dynamically set by openshift_ovirt role
 
 [nodes]
-master0.example.com openshift_node_group_name=node-config-master
-master1.example.com openshift_node_group_name=node-config-master
-master2.example.com openshift_node_group_name=node-config-master
-infra0.example.com openshift_node_group_name=node-config-infra
-infra1.example.com openshift_node_group_name=node-config-infra
-infra2.example.com openshift_node_group_name=node-config-infra
-app0.example.com openshift_node_group_name=node-config-compute
-app1.example.com openshift_node_group_name=node-config-compute
-app2.example.com openshift_node_group_name=node-config-compute
+# dynamically set by openshift_ovirt role
 ----
-<1> If you use a container registry that requires authentication, such as the
-default container image registry, specify the credentials for that account. See
-xref:../install_config/configuring_red_hat_registry.html#install-config-configuring-red-hat-registry[Accessing and Configuring the Red Hat Registry].
-<2>  If you use the `openshift_node_local_quota_per_fsgroup` parameter, you must
-specify the partition or LVM to use for the directory of 
-`/var/lib/origin/openshift.local.volumes`. The partition must be 
-mounted with the option of `gquota` in fstab. 
+
+<1> If you use a container registry that requires authentication, such as the default container image registry, specify the credentials for that account. See xref:../install_config/configuring_red_hat_registry.html#install-config-configuring-red-hat-registry[Accessing and Configuring the Red Hat Registry].
+<2>  If you use the `openshift_node_local_quota_per_fsgroup` parameter, you must specify the partition or LVM to use for the directory of `/var/lib/origin/openshift.local.volumes`. The partition must be mounted with the option of `gquota` in *fstab*.
 +
---
 This inventory file uses the following nodes and disks:
-
++
 * One load balancer instance
```

Comment:
Is this section still accurate? Some of these items appear to be dynamically set now.

Reply:
QE confirmed this.

Reply:
I'd add a note that they're dynamically set by that role.

Reply:
It already says that they're dynamically set by the role -- in the yaml that this callout belongs to.